libs/order/src/models/order-address.ts

```diff
@@ -0,0 +1,30 @@
+import { DaffOrderShippingRate } from './order-shipping-rate';
+import { DaffAddress } from '@daffodil/geography';
+
+export interface DaffOrderAddress extends DaffAddress {
```

Comment:
For now, let's keep this as close to the `DaffAddress` as possible, I think the only thing we really need is `order_id: string | number`.

Reply:
Should it extend from `DaffPersonalAddress` instead?

Reply:
Ah, yes. That's a good call.

Reply:
changed